sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py

```diff
@@ -15,7 +15,49 @@
 # limitations under the License.
 #
 
-"""A word-counting workflow that uses Google Cloud Datastore."""
+"""A word-counting workflow that uses Google Cloud Datastore.
+
+This example shows how to use ``datastoreio`` to read from and write to
+Google Cloud Datastore. Note that running this example may incur charge for
+Cloud Datastore operations.
+
+See https://developers.google.com/datastore/ for more details on Google Cloud
+Datastore.
+See https://cloud.google.com/dataflow/docs/quickstarts/quickstart-python on
```

Comment:
Should this be `http://beam.incubator.apache.org/get-started/quickstart/` instead?